lib/spack/spack/util/executable.py

```diff
@@ -235,7 +233,11 @@ def streamify(arg, mode):
             return result
 
         except OSError as e:
-            raise ProcessError("%s: %s" % (self.exe[0], e.strerror), "Command: " + cmd_line_string)
+            message = "Command: " + cmd_line_string
+            if sys.platform == "win32" and " " in self.exe[0]:
+                message = "\nDid you mean to add a space to the command?"
```

Comment:
Two things:

* you probably want `+=`
* this message should be printed on Linux as well

Reply:
Good catches. Those changes are applied.